src/styles/blocks/catalog.scss

```diff
@@ -11,15 +10,20 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
-
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $speed-of-change;
```

Comment:
Also add [transition-property](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property) rule and indicate exactly which properties will change to make better performanse